javascript-functions-practice/get-character-at-index.js

```diff
@@ -0,0 +1,6 @@
+/* exported getCharacterAtIndex */
+
+function getCharacterAtIndex(string, index) {
+  const char = string.charAt(index);
```

Comment:
This is totally fine, but you can also use brackets just like an array to get a value! So you can do `string[index]`. Either way is fine, but I just wanted to show you another way!